src/components/App/App.tsx

```diff
@@ -936,7 +961,8 @@ export default class App extends React.Component<null, AppState> {
   };
 
   setMedia = (e: any, data: DropdownProps) => {
-    this.socket.emit('CMD:host', data.value);
+    this.socket.emit('CMD:addVideoToPlaylist', data.value);
```

Comment:
Maybe we can just use the existing host command and have the server queue it in a list if there's already something.

Something else to think about: It might also be useful to present "play immediately" and "add to playlist" as two separate actions.

In my own usage I sometimes just want to change the video without queueing it, so it would be good to keep that ability.

Reply:
Yes, I'd keep them separate since people would maybe still directly queue a video in front of the playlist

Reply:
Isn't this still replacing the setMedia logic?

Should we have a separate function that sends addVideoToPlaylist and let the server handle popping the first item off the queue when the next video is ready to be played?